### ISSUE TITLE
Vis pris-komponenter på graf/søjlediagram

### ISSUE THREAD

**Issue by hekner** [rndfm/elspotpris#66]
Først noget ros: Fantastisk hjemmeside - mange tak!

Lige nu vises timepriserne som en søjle med samme farve, kunne time-søjlen brydes op i forskellige vertikale komponenter, således at man nemt kan se hvor meget de enkelte ting udgør?

Måske skal man ikke have alle 5-10 komponenter vist med separat farve i timesøjlerne, men f.x. at se transporten som separat del-farve i hver søjle ville være super-relevant, specielt her efter 1. januar 2023 hvor taksterne stiger helt vildt. Så fx. tre farver i hver søjle:
1) Basal elpris (incl. moms) - denne vises nederst i søjlen
2) Transport (incl. moms) - midt i søjlen
3) Elafgift (incl. moms) - øverst i søjlen

Hvis man viste det på denne måde med elafgiften øverst som egen farve kunne man også fjerne fluebenet for elafgift - det vil nu være overflødigt.

**Comment by rndfm** [comment]
Det er en rigtig fin ide.

Kunne det være detalje visning for en enkle søjle hvis man holder over den/trykker på den?

**Comment by hekner** [comment]
Ja, det ville være rigtig godt. Så måske tre farver/inddelinger i søjen som foreslået, og fulde detailler for præcis denne time hvis man trykker / hover over søjlen?

**Comment by rndfm** [comment]
Grunden til at jeg gerne ville "gemme" farveladen lidt er at siden gerne må bevæge sig i en retning hvor den fremstår simpel for folk som ikke er dybt inde i elpris sammensætning. Men hvor man kan grave sig længere ned i informationen hvis man ønsker dette.

**Comment by hekner** [comment]
Så skal "farvelade" eller "ikke-farvelade" valget måske laves som flueben? Fordi man er absolut interesseret i at kunne se pris-breakdown hen over hele døgnet, og ikke kun for den time-søjle som man klikker eller peger på?

**Comment by rndfm** [comment]
God ide. :)
